src/Loading/babylon.sceneLoader.ts

```diff
@@ -71,34 +82,40 @@
             }
 
             var extension = sceneFilename.substring(dotPosition, queryStringPosition).toLowerCase();
+            return SceneLoader._registeredExtensions[extension];

```

Comment:
what if the extension is unknown? We should return the first plugin in this case